CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The 'salary' field is a leftover debug or test line, not following naming conventions or access modifiers. It serves no purpose and should be removed before merging.

```suggestion
// public string salary;  // Removed debug field
```